share/picongpu/tests/CollisionsBeamRelaxation/include/picongpu/param/particle.param

```diff
@@ -91,38 +100,56 @@ namespace picongpu
                 static constexpr float_64 temperature = SI::ELECTRON_MASS_SI * SI::SPEED_OF_LIGHT_SI
                     * SI::SPEED_OF_LIGHT_SI * UNITCONV_Joule_to_keV * 0.00002;
             };
-            using AddTemperatureElectrons = unary::Temperature<TemperatureParamElectrons>;
+            /** Definition of manipulator assigning a temperature
+             *  using parameters from struct TemperatureParamIons.
+             */
             using AddTemperatureIons = unary::Temperature<TemperatureParamIons>;
 
-            struct IonCharge1Impl
+
+            /** Define initial ion charge to once ionized*/
+            struct OnceIonizedImpl
             {
+                /** Ionize ions once by removing one bound electron */
                 template<typename T_Particle>
                 DINLINE void operator()(T_Particle& particle)
                 {
-                    particle[boundElectrons_] = 5.0_X;
+                    constexpr float_X protonNumber = GetAtomicNumbers<T_Particle>::type::numberOfProtons;
+                    particle[boundElectrons_] = protonNumber - 1.0_X;
                 }
             };
+            /** Definition of manipulator that sets the ionization state of an ion macro-particle
+             * according properties defined in struct OnceIonizedImpl.
+             */
+            using IonCharge1 = generic::Free<OnceIonizedImpl>;
+
 
-            struct IonCharge3Impl
+            /** Define initial ion charge to thrice ionized*/
+            struct ThriceIonizedImpl
             {
+                /** Ionize ions thrice by removing three bound electrons */
                 template<typename T_Particle>
                 DINLINE void operator()(T_Particle& particle)
                 {
-                    particle[boundElectrons_] = 3.0_X;
```

Comment:
How did this work before?

Reply:
Okay - it's Carbon 6 - 3 = 3

Reply:
just as a discussion point, we have a specific [functor](https://github.com/ComputationalRadiationPhysics/picongpu/blob/f1f9a2d5781a53469a2124dbaa5872a9e8aaba49/include/picongpu/particles/atomicPhysics/SetToAtomicGroundStateForChargeState.hpp#L24-L33) for that initialization.
Used as follows:
```
namespace picongpu::particles::manipulators
{
    struct SetIonTo16Ionized
    {
        /** set ions to targetChargeState times ionized and atomic ground state for
         *  that charge state.
         */
        template<typename T_Particle>
        DINLINE void operator()(T_Particle& particle)
        {
            constexpr float_X targetChargeState = 16;

            // number of Electrons for neutral atom
            constexpr float_X numberBoundElectrons
                = picongpu::traits::GetAtomicNumbers<T_Particle>::type::numberOfProtons;

            PMACC_ASSERT_MSG(
                numberBoundElectrons >= targetChargeState,
                "too few protons for target charge state");

            picongpu::particles::atomicPhysics::SetToAtomicGroundStateForChargeState{}(
                particle,
                numberBoundElectrons - targetChargeState);
        }
    };

    using SetIonization = generic::Free<SetIonTo16Ionized>;

} // namespace picongpu::particles::manipulators
```
do we want to switch to that?

Reply:
Included assert and usage of functor as Brian suggests